modules/mobile/spec/support/vcr_cassettes/providers/get_provider_200.yml

```diff
@@ -0,0 +1,238 @@
+---
```

Comment:
This was copied from the VAOS cassettes and modified. I needed to add a few different provider ids to cover all the cases in ruben's appointments data, so you'll see that there are four requests here and the only differences are the provider ids.